### PR TITLE
Move app.py to package_update.py

### PR DESCRIPTION
**Pull request by KPostOffice**
## Related Issues and Dependencies

Faust app needs to be run via app.sh so to avoid confusion we will change the name of app.py

## This introduces a breaking change

- [ ] Yes
- [ ] No

## This Pull Request implements

… Explain your changes.

### CODE REVIEW

**Comment by goern** [comment]
If you rename this one, don’t we need to change some deployment/cronjob?!

**Comment by KPostOffice** [comment]
I added app.sh, I'll make that change here instead.

**Comment by KPostOffice** [comment]
@goern PTAL

File: app.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env sh
+#
```
Comment (goern) [inline app.sh]: Can’t we set an ENV VAR to point to that file?

Comment (KPostOffice) [reply app.sh]: How do you mean?
Like `SUBCOMMAND` in the other files?
```bash
if SUBCOMMAND==producer
    faust ....
fi
```

Comment (fridex) [reply app.sh]: I think the reference here is to `APP_SCRIPT`:

https://github.com/sclorg/s2i-python-container/tree/master/3.6#environment-variables

**Pull request merged**